### PR TITLE
improve log for debug in case of error

### PR DESCRIPTION
**Pull request by skyao**
# Description

Improve log for debug in case of error: don't eat it, at least write the error into log.

## Issue reference

#2109 

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [x] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by skyao** [comment]
Hi, any feedback?

File: pkg/injector/pod_patch.go [hunk 1]
```diff
@@ -230,6 +230,7 @@ func getTrustAnchorsAndCertChain(kubeClient *kubernetes.Clientset, namespace str
 func mTLSEnabled(daprClient scheme.Interface) bool {
 	resp, err := daprClient.ConfigurationV1alpha1().Configurations(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
 	if err != nil {
+		log.Errorf("Fail to load dapr configuration from k8s, use default value %t for mTLSEnabled: %s", defaultMtlsEnabled, err)
```
Comment (yaron2) [inline pkg/injector/pod_patch.go:233]: nit: `failed to load`

File: pkg/injector/pod_patch.go [hunk 2]
```diff
@@ -238,6 +239,7 @@ func mTLSEnabled(daprClient scheme.Interface) bool {
 			return c.Spec.MTLSSpec.Enabled
 		}
 	}
+	log.Infof("Dapr system configuration is not found, use default value %t for mTLSEnabled", defaultMtlsEnabled)
```
Comment (yaron2) [inline pkg/injector/pod_patch.go:242]: Can you include the name of the configuration here?

**Review by yaron2 — CHANGES_REQUESTED** [review]

**Review by yaron2 — APPROVED** [review]
lgtm

**Pull request merged**